docs/seps/sep-0005.rst

```diff
@@ -1 +1,89 @@
+================================
+SEP 5 — SDyPy Unified timeseries
+================================
+
+:Authors: Wout Weijtjens <wout.weijtjens@vub.be>, 
+:Status: Draft
+:Type: Process
+:Created: 2023-07-07
+
+
+Abstract
+--------
+
+To simplify the interface across packages a unified way to represent timeseries data is proposed. This proposal targets future input-output packages to return the data in this common style.
+
+Motivation and Scope
+--------------------
+
+SDyPy is supporting individual user(s) driven packages. However, for ease of integration, reduce the risk of errors and long-term stability a unified way to represent timeseries data is enforced.
+
+The current proposal targets timeseries data in particular, whether it is for input and or output. A simple but generalistic structure is put forward to allow sufficient flexibility
+for various applications, yet guaranteeing a consistent handling of the data. 
+
+Future packages that provide import/export functionality for data from and to SDyPy will have to comply with this format. A package to test compliance with the standard is to be developed. 
+
+
+Detailed description
+--------------------
+The current proposed structure is that of a list of dictionaries. In which each item in the list represents a single channel. 
+
+
+.. code-block::
+
+  [
+    {
+         "data": np.Array([1,2,3,4]) ,   # 1D array of data
+         "fs": 125,                               # Sampling frequency in Hz
+         "time": np.Array([1,3,4,7]),    # Sampling freq is required for equally spaced data, while `time` for non-equally
+         "name": "channel_id_1",        # channel name
+         "start_timestamp" : "2023/06/29 15:00:00+00:00", # Optional: Timestamp in absolute time
```

Comment:
Should we give more requirements for the formating of timestamp? Obviously, it is a string, but what format? Should it be read by `np.datetime64` (https://numpy.org/doc/stable/reference/arrays.datetime.html, supports nano second accuracy) or by native Python `datetime` (this support time-zones)?
Or should we have a Python timestamp as the number stored (not string)?

Reply:
I typically use the ``datetime``.  Not a fan of timestamps as numbers as they are not human readable.

Reply:
Yes, I completely agree; I would also go with a human-readable format, but then we have to have the default format (https://docs.python.org/3/library/datetime.html#datetime.datetime.strptime) which is [year]-[month]-[day]T[hour]:[minutes]:[seconds].[milliseconds]: `1900-01-01T00:00:00.000`.
In case the format is custom, I would suggest that we also have a `start_timestamp_format` item.
I will make a SEP change suggestion.